build/git/build.sh

```diff
@@ -73,6 +73,9 @@ install_man() {
     popd > /dev/null
 }
 
+TESTSUITE_FILTER='^(#|\*)'
```

Comment:
On second thoughts, ditch this filter otherwise we can't tell which tests have changed status between versions.